src/browser/main.js

```diff
@@ -1,13 +1,11 @@
-'use strict'
-
 import Banner from '../components/banner.svelte'
 
-function attachBanner (target, data = {}) {
+function attachBanner(target, props = {}) {
   const banner = new Banner({
     target,
-    data
+    props
```

Comment:
ah yes - this is a bad port from v2 on my behalf, that would be why it wasn't working for James. I use it with Svelte so in my case, this `main.js` is never used.

Reply:
Yes, it worked for me in Svelte too so I didn't really bother until I saw his issue. Not sure why I picked it up other than wanting to get rid of styles to be honest.